src/Carnac/UI/PreferencesViewModel.cs

```
@@ -87,16 +107,11 @@ public void Visit()
             }
             catch //I forget what exceptions can be raised if the browser is crashed?
             {
-
+                // ignored
```

Otherwise we get a warning about empty catch blocks